CHANGES.md

```diff
@@ -1,3 +1,18 @@
+# Changes in git
+
+* Fix importing schema files by using a relative path.
+* Fix `related_dataset_schema_ids` to also detect changes in nested objects.
+* Fix `DatasetTableSchema.get_fields()` to return cached instances too.
+* Fix performance of iterating over subfields, no longer needs to load related tables.
+* Added `DatasetFieldSchema.is_nested_object` property.
+* Normalized exceptions for missing datasets/tables/fields:
+  * The `DatasetNotFound` exception extends from `SchemaObjectNotFound`.
+  * Added `DatasetTableNotFound` and `DatasetFieldNotFound`.
+  * There is no need for `except (DatasetNotFound, SchemaObjectNotFound)` code, it can all be `except SchemaObjectNotFound:`.
+* Cleanup Django model field creation logic.
+* Cleanup SQLAlchemy column creation logic.
```

Comment:
Zullen we de changelog beperken tot user-visible wijzigingen?

```suggestion
* Cleanup Django model field creation logic.
* Cleanup SQLAlchemy column creation logic.
```

Reply:
goede vraag. daarom stond deze ook onderaan. Het is meer ter info dat als er iets stuk gaat, je iig kan zien welke release een refactor heeft gehad die daar wellicht de oorzaak van is.